### PR TITLE
feat(monitoring): add Prometheus sample

### PR DESCRIPTION
**Pull request by yuriatgoogle**
feat: provides code for documentation.

### CODE REVIEW

**Comment by yuriatgoogle** [comment]
@tbpg Hi, Tyler - thank you very much for your help with the OpenCensus sample.  Could you help me figure out what I need to do to get the Build and Lint check to pass?

File: monitoring/prometheus/main.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	}()
+	// [END monitoring_sli_metrics_prometheus_latency]
+	// [START monitoring_sli_metrics_prometheus_counts]
+	requestCount.Inc()
+
+	// Fails 10% of the time.
+	if rand.Intn(100) >= 90 {
+		log.Printf("intentional failure encountered")
+		failedRequestCount.Inc()
+		// [END monitoring_sli_metrics_prometheus_counts]
```
Comment (tbpg) [inline monitoring/prometheus/main.go:71]: This END seems out of place -- the `if` opening brace `{` doesn't have a matching `}`.

**Review by tbpg — COMMENTED** [review]
Looks good! One small region tag comment.

File: monitoring/prometheus/main.go (hunk 1 above)
Comment (yuriatgoogle) [reply monitoring/prometheus/main.go]: Fixed, thank you.

**Review by tbpg — CHANGES_REQUESTED** [review]
Chatted offline about a couple more changes.

**Pull request merged**